src/uct/cuda/cuda_copy/cuda_copy_ep.c

```diff
@@ -48,31 +48,23 @@ uct_cuda_copy_get_stream(uct_cuda_copy_iface_t *iface,
                          ucs_memory_type_t src_type,
                          ucs_memory_type_t dst_type)
 {
-    cudaStream_t *stream;
+    cudaStream_t *stream = NULL;
     ucs_status_t status;
 
-    if ((src_type == UCS_MEMORY_TYPE_LAST) &&
-        (dst_type == UCS_MEMORY_TYPE_LAST)) {
-        stream = &iface->short_stream;
-    } else if ((src_type < UCS_MEMORY_TYPE_LAST) &&
-               (dst_type < UCS_MEMORY_TYPE_LAST)) {
+    if ((src_type < UCS_MEMORY_TYPE_LAST) &&
+        (dst_type < UCS_MEMORY_TYPE_LAST)) {
```

Comment:
i think we can assert that src_type and dst_type are < LAST since they should be valid types here